tidy3d/components/medium.py

```diff
@@ -108,28 +206,91 @@ def _validate_numiters(cls, val):
         return val
 
 
-class NonlinearSusceptibility(NonlinearSpec):
-    """Specification adding an instantaneous nonlinear susceptibility to a medium.
-    The expression for the instantaneous nonlinear polarization is given below.
+class TwoPhotonAbsorption(NonlinearModel):
+    """Model for two-photon absorption (TPA) nonlinearity which gives an intensity-dependent
+    absorption of the form :math:`\\alpha = \\alpha_0 + \\beta I`.
+    The expression for the nonlinear polarization is given below.
 
     Note
     ----
     .. math::
 
-        P_{NL} = \\epsilon_0 \\chi_3 |E|^2 E
+        P_{NL} = -\\frac{c_0^2 \\varepsilon_0^2 n_0 \\operatorname{Re}(n_0) \\beta}{2 i \\omega} |E|^2 E
 
     Note
     ----
-    The nonlinear constitutive relation is solved iteratively; it may not converge
-    for strong nonlinearities. Increasing `numiters` can help with convergence.
+    This frequency-domain equation is implemented in the time domain using complex-valued fields.
 
     Note
     ----
-    For complex fields (e.g. when using Bloch boundary conditions), the nonlinearity
-    is applied separately to the real and imaginary parts, so that the above equation
-    holds when both E and :math:`P_{NL}` are replaced by their real or imaginary parts.
-    The nonlinearity is only applied to the real-valued fields since they are the
-    physical fields.
+    Different field components do not interact nonlinearly. For example,
+    when calculating :math:`P_{NL, x}`, we approximate :math:`|E|^2 \\approx |E_x|^2`.
+    This approximation is valid when the E field is predominantly polarized along one
+    of the x, y, or z axes.
+
+    Example
+    -------
+    >>> tpa_model = TwoPhotonAbsorption(beta=1)
+    """
+
+    beta: Complex = pd.Field(
+        0,
+        title="TPA coefficient",
+        description="Coefficient for two-photon absorption (TPA).",
+        units="um / W",
```

Comment:
let's use the strings for constants from `constants.py` and just insert them here. Reason is if we ever want to change how they are displayed, we can do it globally by just changing `constants.py`.

Reply:
ok, I had to add WATT and VOLT to this file, hopefully that's ok

Reply:
yea sounds good